java/.ci/Jenkinsfile

```diff
@@ -919,6 +919,31 @@ pipeline {
                          }
                      }
                 }
+                stage('KAFKA TO GCS via Dstream (bytes)'){
+                     steps {
+                         retry(count: stageRetryCount) {
+                             sh '''
+
+                                  export SKIP_BUILD=true
+                                  cd java
+                                  bin/start.sh \
+                                  -- \
+                                  --template KafkaToGCSDstream \
+                                  --templateProperty project.id=$GCP_PROJECT \
+                                  --templateProperty kafka.bootstrap.servers=$ENV_TEST_KAFKA_BOOTSTRAP_SERVERS \
+                                  --templateProperty kafka.topic=integration-test \
+                                  --templateProperty kafka.starting.offset=latest \
+                                  --templateProperty kafka.gcs.output.location=gs://dataproc-templates/integration-testing/output/KAFKATOGCS/bytes \
```

Comment:
Change path `KAFKATOGCS/bytes` to something like `KAFKATOGCS_DStream/parquet`, to avoid future conflicts when we expand the integration test cases for our existing templates

Reply:
Done